src/main/java/com/minipulse/db/MiniPulseDBImpl.java

```diff
@@ -60,7 +64,7 @@ public void savePollAndOverwriteQuestions(Poll poll) {
         String insertMultipleChoiceQuestionQuery = "INSERT INTO multiple_choice_questions(question_id, poll_id, question_text, question_desc, is_mandatory) VALUES (?, ?, ?, ?, ?)";
```

Comment:
we have some missing tables that we need to implement. The question table where the text question, MCA and SCQ questions inherit.

this goes the same for answers as well. Please refer to the Table Schema and try to code after understanding it.